packages/insured-bridge-relayer/src/index.ts

```diff
@@ -170,14 +171,16 @@ export async function run(logger: winston.Logger, l1Web3: Web3): Promise<void> {
               if (config.botModes.l2FinalizerEnabled) await crossDomainFinalizer.checkForBridgeableL2TokensAndBridge();
               else logger.debug({ at: "AcrossRelayer#CrossDomainFinalizer", message: "L2->L1 finalizer disabled" });
 
+              // The multicall bundler likely accrued transactions over the course of the run.
+              // This call fires off those transactions, but does not wait on them to be mined.
+              // Note: we wait until this point to actually send off the transactions to
+              await multicallBundler.send();
```

Comment:
Doesn't this batch transactions per relayer network? So All arbitrum relays + settles are batched, and on the next run, all optimism ...

Reply:
You're totally right. I merged this incorrectly with the PR that added multichain support.

Reply:
Fixed.